tests/xtimer_msg/Makefile

```diff
@@ -0,0 +1,8 @@
+APPLICATION = xtimer_msg
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_RAM := stm32f0discovery
```

Comment:
Building for stm32f0discovery works on my machine, can you try removing this and see if Travis fails?